examples/plotting/plot_modify_colorbar.py

```diff
@@ -0,0 +1,119 @@
+import cartopy.crs as ccrs
+
+import pyart
+from pyart.testing import get_test_data
+
+#########################################
+# ** Initial PPI Map Display 
+#
+# Let's plot a PPI Map Display 
+# and take a look at the colorbar
+#
+# Notice: the colorbar is not perfect 
+# and slighltly overlaps the PPI display
+
+# Define figure
+fig = plt.figure()
+
+# Create a subplot with correct cartopy projection
+axs = plt.subplot(111, projection=ccrs.PlateCarree())
+
+# Define and Read in the test data
+radar_file = get_test_data('swx_20120520_0641.nc')
+radar = pyart.io.read(radar_file)
+
+# Create the Radar Map Display (defines x,y as lat/lons)
+display = pyart.graph.RadarMapDisplay(radar)
+
+# Display the horizontal equivalent reflectivity factor
+# Note: embellish = False will remove initial lat/lon coords
+display.plot_ppi_map('reflectivity_horizontal',
+                      2,
+                      ax=axs,
+                      vmin=-30,
+                      vmax=60,
+                      embellish=False,
+                      norm=None,
+                      cmap='pyart_HomeyerRainbow')
+
+# Add gridlines
+gl = axs.gridlines(crs=ccrs.PlateCarree(),
+                   draw_labels=True,
+                   linewidth=1, 
+                   color='gray', 
+                   alpha=0.3,
+                   linestyle='--')
+
+plt.gca().xaxis.set_major_locator(plt.NullLocator())
+
+# Make sure labels are only plotted on the left and bottom
+gl.top_labels = False
+gl.right_labels = False
+
+gl.xlabel_style = {'size': 14}
+gl.ylabel_style = {'size': 14}
+
+############################################
+# ** Colorbar Position / Title Manipulation
+
+# Now, let's update the colorbar position
+# to match the display
+
+# Define figure
+fig = plt.figure()
+
+# Create a subplot with correct cartopy projection
+axsB = plt.subplot(111, projection=ccrs.PlateCarree())
+
+# Create the display again
+# Note: colorbar_flag is set to False
+#   to not initally create the colorbar. 
+display.plot_ppi_map('reflectivity_horizontal',
+                      2,
+                      ax=axsB,
+                      vmin=-30,
+                      vmax=60,
+                      embellish=False,
+                      norm=None,
+                      cmap='pyart_HomeyerRainbow',
+                      colorbar_flag=False)
+
+# Add gridlines
+gl = axsB.gridlines(crs=ccrs.PlateCarree(),
+                   draw_labels=True,
+                   linewidth=1, 
+                   color='gray', 
+                   alpha=0.3,
+                   linestyle='--')
+
+plt.gca().xaxis.set_major_locator(plt.NullLocator())
+
+# Make sure labels are only plotted on the left and bottom
+gl.top_labels = False
+gl.right_labels = False
+
+gl.xlabel_style = {'size': 12}
+gl.ylabel_style = {'size': 12}
+
+# Define the colorbar from the RadarMapDisplay object
+#Modify the colorbar title and size
+cbar = plt.colorbar(mappable=display.plots[0], fraction=0.04, ax=axsB)
```

Comment:
Should you be able to set cbar = display.cbs[0]? This is usually how I modify it.. display.cbs should return a list of colorbar objects